lib/resources/majors.tsx

```diff
@@ -173,34 +178,39 @@ const baCogSci: Major = {
             'At least 2 courses must be at the 300 level or above.',
           required_credits: 12,
           criteria: 'COGS-COGPSY[T]',
+          double_count: ['All'],
```

Comment:
On the Major schema in the backend, lets make this the default value for double_count for Major documents.

Reply:
in backend yes